testcontainers/src/core/container.rs

```diff
@@ -168,20 +168,30 @@ where
             })
     }
 
-    /// Returns the bridge ip address of docker container as specified in NetworkSettings.IPAddress
+    /// Returns the bridge ip address of docker container as specified in NetworkSettings.Networks.IPAddress
     pub fn get_bridge_ip_address(&self) -> IpAddr {
         IpAddr::from_str(
             &self
                 .docker_client
                 .inspect(&self.id)
                 .network_settings
-                .unwrap_or_default()
-                .ip_address
+                .and_then(|settings| {
+                    settings
+                        .networks
+                        .zip(self.image.network().clone().or(settings.bridge))
+                })
+                .and_then(|(mut networks, bridge_name)| networks.remove(&bridge_name))
+                .and_then(|net| net.ip_address)
                 .unwrap_or_default(),
         )
         .unwrap_or_else(|_| panic!("container {} has missing or invalid bridge IP", self.id))
     }
 
+    /// Returns the inspect result of docker container
+    pub fn inspect(&self) -> ContainerInspectResponse {
+        self.docker_client.inspect(&self.id)
+    }
```

Comment:
Please remove this part. We are not going to add `ContainerInspectResponse` to our public API.

Reply:
Ok, no-problem. I re-exported it cause it was asked in another PR.